openmaptiles/styleutils.py

```diff
@@ -0,0 +1,86 @@
+import json
+from pathlib import Path
+from openmaptiles.tileset import Tileset
+
+
+def fp_to_dict(fp: str) -> dict:
+    with open(fp, 'r') as f:
+        d = json.load(f)
+    return d
+
+
+def get_ts_lyr_style_json_fp(yaml_fp: str) -> Path:
+    yaml_path = yaml_fp.filename
+    json_name = Path(yaml_path.name).with_suffix('.json')
+    ts_lyr_dir = yaml_path.parent
+
+    return ts_lyr_dir.joinpath(json_name)
+
+
+def add_order(lyrs: list) -> list:
+    for i in range(0, len(lyrs)):
+        lyr = lyrs[i]
+        lyr['order'] = i
```

Comment:
you want to use enumerate:

```suggestion
    for i, lyr in enumerate(lyrs):
        lyr['order'] = i
```

Reply:
Applied.